.github/workflows/smoke-tests-canaries.yml

```diff
@@ -0,0 +1,17 @@
+name: Smoke Tests - Canaries
```

Comment:
What's the notification mechanism for this canary? I.e. how would the oncall find out if a run has failed?

Reply:
This is a good question.

Options to pursue:
- (lightweight) Have a slack bot that listens to this workflow, see https://github.blog/changelog/2022-12-06-github-actions-workflow-notifications-in-slack-and-microsoft-teams/ .
- (more involved) Add an action that publishes metric to CloudWatch like our friends did here https://github.com/aws-amplify/amplify-ui/blob/a787311029e6c0c179b552f5abdad1dbed73436a/.github/actions/log-metric/action.yml#L43 .

I'd try that Slack integration first after merging this and explore cloudwatch if this is not working.